### PR TITLE
Feat : added cart page

### PR DESCRIPTION
**Pull request by SRUJANA-PENUGONDA13**
### Added Cart Page
cart

### CODE REVIEW

File: css/components/cart.css [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+.price-details-item {
+  justify-content: space-between;
+}
+
+.line-divider {
+  border: 0.2px solid var(--primary--color);
+}
+
+.cart-container .primary-btn {
```
Comment (irajbhinde) [inline css/components/cart.css:48]: You can add cursor : pointed to your button

**Review by irajbhinde — COMMENTED** [review]
1) You're missing out on lots of cursor pointers & the anchor links, user should be able to browse through different screens. For example nav title doesn't have an anchor link which takes use to home page. 

2) The position of the trending badge in product listing could be improved. My suggestion would be to take it to the extreme left of card *since* you are already using the extreme top of it.

Otherwise the website looks great, keep up the good work.

**Review by Rishi05051997 — APPROVED** [review]
Looks Good for me

**Comment by SRUJANA-PENUGONDA13** [comment]
Thanks @irajbhinde @Rishi05051997 for reviewing

**Pull request merged**